source/contribute/website.rst

```diff
@@ -140,6 +115,37 @@ After the initial build you can watch for frontend asset changes to update the s
 
 This process will run in the foreground and recompile the assets when a change is made to them. After refreshing the browser you should see the new assets loaded.
 
+Run tests
+---------
+
+The Doctrine website includes Unit Tests and some Integration Tests to cover its functionality and to keep it stable.
+
+JavaScript
+~~~~~~~~~~
+
+If some changes are provided for JavaScript then there have to be test written in `Jest <https://jestjs.io>`. You'll
+find the Jest tests in the ``jest`` directory of the Doctrine website project. The tests can be run with the following
+command:
+
+.. code-block:: console
+
+    $ yarn jest
+
+PHP
+~~~
+
+PHP tests are using `PHPUnit <https://phpunit.de>` to cover its code. If you want to run tests for PHP, you have to
```

Comment:
"to cover _its_ code"? :thinking: What does "its" refer to here?

Reply:
I want to cover the code of PHPUnit in the Doctrine website project. Makes a lot of sense, right? RIGHT? :sweat_smile: 

Fixed. :smile:

Reply:
Hahaha yeah of course, what was I thinking?